cmd/resource/get.go

```diff
@@ -70,6 +70,25 @@ var (
 			}
 			return service
 		},
+		"route": func(id sdk.ID, options map[string]any) interface{} {
```

Comment:
I considered it for a while. Why not just let then handler reading the configuration directly, then we don't need to change the handler prototype.

Reply:
okey, fixed